pages/linux/ifdown.md

```diff
@@ -1,6 +1,7 @@
 # ifdown
 
 > Disable network interfaces.
+> More information: <https://linux.die.net/man/8/ifdown>.
```

Comment:
I think generally we're moving to man7.org links here, but this is a very interesting case where linux.die.net has the man page, but man7.org doesn't.

`at` and `ip` do (and I see this PR correctly links to those), but `ifup` and `ifdown` don't.

I can't remember why we prefer man7 over linux.die.net though.

Reply:
#5143. Per @mebeim:

>[man7.org] IMO renders the pages a lot better (sometimes die.net pages have syntax errors due to special characters being treated strangely when rendered in HTML, furthermore man7.org pages are actually in monospace).

Reply:
Let's just leave it for now.  I'll update it later perhaps with the rest of them (if I can find an alternative.)

Reply:
@sbrl : yes that's why I put the die.net links here because I could not find them in man7.org ^^